pkg/http.go

```diff
@@ -4,19 +4,20 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
+	"io"
 	"net/http"
 	"strings"
 
 	"github.com/gookit/color"
 
-	"github.com/skuid/tides/pkg/constants"
-	"github.com/skuid/tides/pkg/errors"
-	"github.com/skuid/tides/pkg/logging"
+	"github.com/skuid/skuid-cli/pkg/errors"
+	"github.com/skuid/skuid-cli/pkg/logging"
 )
 
+var VERSION_NAME string
+
 var (
-	SkuidUserAgent = fmt.Sprintf("Tides/%s", constants.VERSION_NAME)
+	SkuidUserAgent = fmt.Sprintf("skuid-cli/%s", VERSION_NAME)
```

Comment:
This variable is being initialized before Run() command in main, because main.go is importing the packet before cmd.VERSION_NAME or pkg.VERSION_NAME is being set
![image](https://user-images.githubusercontent.com/30167939/205173682-320f3a1d-d41a-4bb4-afa3-deafcd9cbc81.png)

Reply:
Ah, good call. I need to move that to an `init()`, I think.

Reply:
I did a quick test and I set it in the retrieve method but I'm getting an error of Invalid header 

`Post "https://guranora.pliny.webserver:3000/auth/oauth/token": net/http: invalid header field value "skuid-cli 0.6.0\n" for key User-Agent`

![image](https://user-images.githubusercontent.com/30167939/205184697-01d6905d-364d-45b9-8411-ce9fe34f2125.png)

Reply:
Indeed, whitespace can't be in there. Now that it's initialized for those packages in init() I made sure to trim it before the user agent string is constructed.